package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "agenda",
-  "version": "4.2.1",
+  "version": "4.3.1",
```

Comment:
please don't do a version push in the PR itself, this will be done on release itself

Reply:
Done